src/Form.ts

```diff
@@ -152,8 +152,8 @@ export default class Form extends Base implements FormInterface {
     return this.check(FieldPropsEnum.disabled, true);
   }
 
-  makeField(data: FieldConstructor) {
-    return new Field(data);
+  makeField(data: FieldConstructor, FieldClass: new (data: FieldConstructor) => Field = Field) {
```

Comment:
Just using `Field` instead of `new (data: FieldConstructor) => Field`
have you tried?